web/gui/src/main/resources/templates/logging.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <title>Log in to Yggdrasil Library Robot</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="../css/reset.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    </head>
+<body>
+
+<div class="logging">
```

Comment:
I am not sure of the usage of "for" in the label, from what I can see it will submit it with mapping usermail - provided email; password - provided password . Still IntelliJ is telling me about invalid id reference sooo I haven't got to this part yet :D What I need to provide?